core/modules/wmgr/SConscript

```diff
@@ -0,0 +1,10 @@
+import pytarget
```

Comment:
it seems we have two tools to manage python modules: 
- site_scons/pytarget.py
and
- site_scons/site_tools/pymod.py (grep for InstallPythonModule)